gradle/libs.versions.toml

```diff
@@ -28,7 +28,7 @@ xmlutil-serialization = 'io.github.pdvrieze.xmlutil:serialization:0.84.2'
 
 dokkaPlugin = "org.jetbrains.dokka:dokka-gradle-plugin:1.7.10"
 spotlessPlugin = "com.diffplug.spotless:spotless-plugin-gradle:6.8.0"
-gradleMavenPublishPlugin = "com.vanniktech:gradle-maven-publish-plugin:0.21.0"
+gradleMavenPublishPlugin = "com.vanniktech:gradle-maven-publish-plugin:0.22.0"
```

Comment:
[Testing publishing a snapshot](https://github.com/cashapp/redwood/actions/runs/3449028112/jobs/5756588155) to ensure everything still works.